### PR TITLE
[DOCS] Standardize language around GX Cloud access tokens

### PR DESCRIPTION
**Pull request by cdkini**
Changes proposed in this pull request:
- Standardize language around tokens

### Definition of Done
- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by ghost** [comment]
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=7621&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=7621&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map legend
</details>

File: docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md [hunk 1]
```diff
@@ -26,16 +26,16 @@ Once you have completed this guide you will have a foundation in the basics of u
...
 
-</Prerequisites> 
+</Prerequisites>
 
 ## Steps
 
 ### 1. Setup
 
-#### 1.1 Generate User Token
+#### 1.1 Generate Access Token
```
Comment (cdkini) [inline docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md]:
```suggestion
#### 1.1 Generate access token
```

File: docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md [hunk 2]
```diff
@@ -26,16 +26,16 @@ Once you have completed this guide you will have a foundation in the basics of u
...
 
-Go to [“Settings” > “Tokens”](https://app.greatexpectations.io/tokens) in the navigation panel and generate a User Token. Both `admin` and `editor` roles will suffice for this guide.
-These tokens are view-once and stored as a hash in Great Expectation Cloud's backend database. Once you copy the API key and close the dialog, the Cloud UI will never show the token value again. 
+Go to [“Settings” > “Tokens”](https://app.greatexpectations.io/tokens) in the navigation panel and generate an Access Token. Both `admin` and `editor` roles will suffice for this guide.
```
Comment (cdkini) [inline docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md]:
```suggestion
Go to [“Settings” > “Tokens”](https://app.greatexpectations.io/tokens) in the navigation panel and generate an access token. Both `admin` and `editor` roles will suffice for this guide.
```

File: docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md [hunk 3]
```diff
@@ -53,14 +53,14 @@ import os
 
 #### 1.3 Create Data Context
 
-Paste this snippet into the next notebook cell to instantiate Cloud <TechnicalTag tag="data_context" text="Data Context"/>. 
+Paste this snippet into the next notebook cell to instantiate Cloud <TechnicalTag tag="data_context" text="Data Context"/>.
 
 :::caution
-Please note that GX Cloud API tokens are sensitive information and should not be committed to version control software. Alternatively, add these as [Data Context config variables](https://docs.greatexpectations.io/docs/guides/setup/configuring_data_contexts/how_to_configure_credentials/)
+Please note that user access tokens are sensitive information and should not be committed to version control software. Alternatively, add these as [Data Context config variables](https://docs.greatexpectations.io/docs/guides/setup/configuring_data_contexts/how_to_configure_credentials/)
```
Comment (cdkini) [inline docs/docusaurus/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud.md]:
```suggestion
Please note that access tokens are sensitive information and should not be committed to version control software. Alternatively, add these as [Data Context config variables](https://docs.greatexpectations.io/docs/guides/setup/configuring_data_contexts/how_to_configure_credentials/)
```

**Review by elenajdanova — APPROVED** [review]

**Review by elenajdanova — APPROVED** [review]

**Pull request merged**